packages/tracing/src/browser/metrics.ts

```diff
@@ -170,6 +171,25 @@ export class MetricsInstrumentation {
       }
 
       transaction.setMeasurements(this._measurements);
+
+      if (this._lcpEntry) {
+        logger.log('[Measurements] Adding LCP Data');
+        // Capture Properties of the LCP element that contributes to the LCP.
+
+        if (this._lcpEntry.element) {
+          transaction.setTag('lcp.element', htmlTreeAsString(this._lcpEntry.element));
+        }
+
+        if (this._lcpEntry.id) {
```

Comment:
According to the type you shouldn't need the check here since it is always defined? I'm fine with an extra guard though in case.

Reply:
According to the spec, it's initially set as an empty string; which is falsey in js. It might be better to not set these explicitly to empty strings.

Reply:
Fair enough